app.json

```diff
@@ -14,7 +14,8 @@
     "review": {
       "env": {
         "PANGEO_FORGE_DEPLOYMENT": "pforge-pr-136"
-      }
+      },
+      "addons": ["papertrail:choklad"]
```

Comment:
does this mean this is only enabled for review apps?

Reply:
@andersy005, yes I'm going to start by seeing what this looks like in a review app, but if it works as I'd hope, I will add it to the top-level add-ons before merging this PR.

Reply:
I figured out that you could configure review app add-ons by reading this https://devcenter.heroku.com/articles/github-integration-review-apps#environments-in-app-json